query.js

```diff
@@ -30,13 +32,24 @@ function subfeed(feedId) {
 }
 
 exports.init = function (sbot, config) {
-  return {
+  const self = {
+    /**
+     * Gets the stored seed message (on the main feed) for the root meta feed.
+     *
+     * ```js
+     * sbot.metafeeds.query.getSeed((err, seed) => {
+     *   console.log("seed buffer", seed)
+     * })
+     * ```
+     */
     getSeed(cb) {
       // FIXME: maybe use metafeed id
       sbot.db.query(
         where(and(author(sbot.id), type('metafeed/seed'))),
-        toCallback((err, msgs) => {
+        paginate(1),
```

Comment:
I wonder if we do this paginate(1) that we do a descending. If one for whatever reason posts two of those message the latest should be the correct one.